### DISCUSSION TITLE
Cleaner home page.

### DISCUSSION THREAD

**Discussion by DeadMeat-12** [anthony1x6000/ROBLOX2016stylus#39] [Ideas]
Is it possible for you to make a option to have the homepage only show the favorites and continue? As far as I remember, the cluttering of the home page was more of a recent change, so this shouldn't be too out of the scope of this project.

**Comment by anthony1x6000** [comment]
I've thought of doing that. It's probably possible, it's just it'd be kinda hard cause all the selectors for the game containers are all the exact same -  so with the userstyle I'd have to use nth-child selectors to remove the unwanted containers, meaning if roblox added extra containers for game genres, my script would probably break the home page. 

I'll try doing that later in JavaScript and maybe make it dig into the content of the containers to see if the header matches continue or favorites. 
100% a feature I'd like to implement.

**Reply by DeadMeat-12** [reply]
With the home page getting even more cluttered, is there any progress on this feature?

**Reply by anthony1x6000** [reply]
There is an option in the 1.2.9 release where you can disable games under "Recommended For You" under the displayRecommended variable in the userstyle settings. Accessing this in stylus can be done by pressing the gear icon next to the style name.  
Hiding recommended games leaves only favorites and sponsored. 

However, outside of that, not much is done.
